index.html

```diff
@@ -5,8 +5,124 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class="pege-header">
+      <nav class="navigation">
+          <img src="logo.svg" alt="Город Седона" width="40" height="30">
```

Comment:
```suggestion
          <img src="logo.svg" alt="Город Седона." width="40" height="30">
```

в альтах обязательно ставить точки, чтобы скрин-ридер делал интонационные паузы

Reply:
нужно поправить везде